source/_integrations/meteoalarm.markdown

```diff
@@ -23,7 +23,7 @@ To enable this binary sensor, add the following lines to your `configuration.yam
 ```yaml
 binary_sensor:
   - platform: meteoalarm
-    country: "NL"
+    country: "netherlands"
```

Comment:
Sure it shouldn't be `Netherlands`? I think to see the website using capitalized country names

Reply:
It should be in lowercases, see: https://feeds.meteoalarm.org/feeds/meteoalarm-legacy-atom-netherlands

Reply:
Ok. Maybe nice to catch this in the upstream package? so if someone capitalizes the country, you just make lower_case out of it anyway?

Reply:
Never mind, it's already there:

https://github.com/rolfberkenbosch/meteoalert-api/blob/master/meteoalertapi/meteoalertapi.py#L14